packages/game-app/src/_shared/components/Input/Input.tsx

```diff
@@ -22,6 +22,7 @@ const Input: React.FC<Props> = React.forwardRef<HTMLInputElement, Props>(
           className={className}
           withIcon={!!iconLeft}
           variant={variant}
+          maxLength={100}
```

Comment:
This implies that if not specified, maxLength will be 100. So what about inputs like scenarioContent?